<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>meeting_window</class>
 <widget class="QMainWindow" name="meeting_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>674</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/pics/official_icon.png</normaloff>:/pics/official_icon.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(255,25,25);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="share_screen_btn">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);

color: </string>
    </property>
    <property name="text">
     <string>START SHARE SCREEN</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="username">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>101</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>USERNAME </string>
    </property>
   </widget>
   <widget class="QLabel" name="meeting_id">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>341</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MEETING ID:</string>
    </property>
   </widget>
   <widget class="QLabel" name="meeting_password">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>171</width>
      <height>19</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>MEETING PASSWORD:</string>
    </property>
   </widget>
   <widget class="QLabel" name="phone_symbol">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>0</y>
      <width>81</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/pics/END_MEETING.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="end_meeting_btn">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>20</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="chat_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>391</width>
      <height>461</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QTextEdit" name="msg_text">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>600</y>
      <width>301</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="send_btn">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>600</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(, 139, 141);
background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>SEND</string>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="end_share_btn">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
background-color: rgb(170, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>END SHARE SCREEN</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>true</bool>
    </property>
    <property name="default">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>phone_symbol</zorder>
   <zorder>share_screen_btn</zorder>
   <zorder>username</zorder>
   <zorder>meeting_id</zorder>
   <zorder>meeting_password</zorder>
   <zorder>end_meeting_btn</zorder>
   <zorder>chat_text</zorder>
   <zorder>msg_text</zorder>
   <zorder>send_btn</zorder>
   <zorder>end_share_btn</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
